app.py

```diff
@@ -77,6 +77,14 @@ def croissant_dataset(dataset):
 def citation_dataset(dataset):
     return render_template("citation.cff", dataset=dataset)
 
+@app.get("/sitemap.xml")
+def sitemap():
+    data = load_data()
+    kwargs = {
+        "data": data
+    }
+    rendered_sitemap = flask.render_template("sitemap.xml", **kwargs)
```

Comment:
no need to create the kwargs element, you can just use:
```suggestion
    rendered_sitemap = flask.render_template("sitemap.xml", data=data)
```